tests/helpers/test_selector.py

```diff
@@ -858,22 +858,38 @@ def test_language_selector_schema(schema, valid_selections, invalid_selections)
                     "longitude": 2.0,
                     "radius": 3.0,
                 },
+                {"latitude": 1.0, "longitude": "1.0"},
```

Comment:
Why is the longitude a string?

Reply:
These were just the original testcases, I didn't choose to add this. 

This used to be failure testcase, but now that float coercion is enabled, this is now a passing testcase.

Reply:
What about removing this case, as it is not a failing case. It will remove the need for the custom converter.

Reply:
I'm satisfied with removing it if you are OK with that.